bundles/org.openhab.voice.actiontemplatehli/README.md

```diff
@@ -0,0 +1,336 @@
+
+### Tokenizer
+
+You can provide a custom model at '<OPENHAB_USERDATA>/actiontemplatehli/token.bin', otherwise it will use the build in simple tokenizer or whitespace tokenizer (configurable).
+
+Tokenizing the text is enough to use the action type 'tokens' as tokens are the only ones required for scoring (but the option 'optionalLanguageTags' will not take effect unless you have the POS language tags).
+
+### POSTagger (language tags)
+
+You need to provide a model for POS tagging at '<OPENHAB_USERDATA>/actiontemplatehli/pos.bin' for your language. 
+This will produce a language tag for each token, that can be used in 'optionalLanguageTags' to make some optional for scoring.
+Note that you need the correct language tags for the lemmatizer to work.
+
+### Lemmatizer
+
+You need to provide a model for the lemmatizer at '<OPENHAB_USERDATA>/actiontemplatehli/lemma.bin' for your language. 
+This will produce a lemma for each token, now you can use the action type 'lemmas'
+
+## Action Scoring
+
+The scoring mechanism scores each action config to look for the best scored.
+If some token fails the comparison the action scores 0.
+If all action scored 0 none is executed.
+
+By the value of the field 'type' it defines if the template should by compared with the tokens or the lemmas.
+Please note that the captured placeholder value is extracted from the tokens not from the lemmas, but the equivalent lemmas are replaced before scoring.
+
+The action template string is a list of tokens separated by white space.
+You can use the ';' separator to provide alternative templates and the '|' to provide alternative tokens.
+
+Take in account that, as this is a token basis comparison, matching changes depending on the tokenizer you are using as they can produce different tokens for the same text.
+
+### Interpreter Configuration
+
+| Config   |  Group  |  Type  |   Default  | Description  |
+|----------|----------|----------|----------|------------------------------|
+| lowerText | nlp | boolean | false | Convert the input text to lowercase before processing |
+| caseSensitive | nlp | boolean | false | Enable case sensitivity, do not apply to dictionaries and models, do not apply to the 'itemLabel' placeholder |
+| useSimpleTokenizer | nlp | boolean | false | Prefer simple tokenizer over white space tokenizer |
+| optionalLanguageTags | nlp | text |  | Comma separated POS language tags that will be optional when comparing |
+| fallbackHLI | nlp | text |  | Human language interpreter to use if command is unhandled |
+| commandSentMessage | messages | text | Done | Message for successful command |
+| unhandledMessage | messages | text | I can not do that | Message for unsuccessful action |
+| failMessage | messages | text | There was an error | Message for error during processing |
+
+## Examples:
+
+### String type action configs example:
+
+This example contain the files to add actions for open an android application and check what is open.
```

Comment:
This example **contains** the files to add actions for **opening** an android application and **checking** what **application** is **opened**.